libfwupdplugin/fu-context.c

```diff
@@ -1367,6 +1368,38 @@ fu_context_add_esp_volume(FuContext *self, FuVolume *volume)
 	g_ptr_array_add(priv->esp_volumes, g_object_ref(volume));
 }
 
+/**
+ * fu_context_set_preferred_esp_path:
+ * @self: The FuContext object.
+ * @path: The path to the preferred ESP.
```

Comment:
If you do `@self: a #FuContext` then it's clickable in the docs -- also no need to start with `The` and no final full-stop.